packages/server/src/test/java/software/uncharted/terarium/hmiserver/service/tasks/TaskServiceTest.java

```diff
@@ -161,6 +161,23 @@ public void testItCanSendGoLLMGenerateSummaryRequest() throws Exception {
 		log.info(new String(resp.getOutput()));
 	}
 
+	// @Test
+	@WithUserDetails(MockUser.URSULA)
+	public void testItCanSendGoLLMGenerateResponseRequest() throws Exception {
+		final TaskRequest req = new TaskRequest();
+		req.setType(TaskType.GOLLM);
+		req.setScript(GenerateResponseHandler.NAME);
+		final GenerateResponseHandler.Input input = new GenerateResponseHandler.Input();
+		input.setInstruction("Give me a simple random json object");
+		final JsonNode resFormat = new ObjectMapper().readTree("{\"type\": \"json_object\"}");
+		input.setResponseFormat(resFormat);
+		req.setInput(input);
+
+		final TaskResponse resp = taskService.runTaskSync(req);
+
+		log.info(new String(resp.getOutput()));
+	}
```

Comment:
I am wondering if we really want to invoke a unit test where the result is inherently stochastic 🤔

Reply:
Just checking if things don't break :)